source/Octopus.Tentacle.Core/Octopus.Tentacle.Core.csproj

```diff
@@ -31,4 +31,13 @@
         <Compile Include="..\Solution Items\VersionInfo.cs" Link="VersionInfo.cs" />
         <Compile Include="..\Solution Items\SolutionInfo.cs" Link="SolutionInfo.cs" />
     </ItemGroup>
+    <ItemGroup>
+      <Folder Include="Util\Extensions\" />
```

Comment:
I'm confused. Why is this here?

Reply:
If rider would stop adding random folder to csproj folders I would be so happy